js/ui/endSessionDialog.js

```diff
@@ -417,6 +447,18 @@ var EndSessionDialog = new Lang.Class({
         this._user.disconnect(this._userChangedId);
     },
 
+    _syncUpdateStatus: function()  {
+        // Start retrieving the Mogwai proxy
+        this._proxy = new SchedulerProxy(Gio.DBus.system,
+                                         'com.endlessm.MogwaiSchedule1',
+                                         '/com/endlessm/DownloadManager1');
```

Comment:
The proxy should be initialized in `_init()`, together with the other proxies, and asynchronously.